gulp/config.js

```diff
@@ -33,7 +33,8 @@ module.exports = {
       'config/**/*.js',
       'gulpfile.js',
       'gulp/**/*.js',
-      'scripts/npm/**/*.js'
+      'scripts/npm/**/*.js',
+      'jest.config.js'
```

Comment:
Where does this get used at?

Reply:
https://github.com/cfpb/cfgov-refresh/blob/master/gulp/tasks/test.js#L61
I really wish it could go in /test/ but I couldn't figure out the paths, since it needs reference things in both /cfgov/unprocessed/ and /test/ from the root directory.